src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
😆Okay here are some opinions: 

I definitely agree that the trailing slash on last items should always be removed but if there's a chance that devs or designers want to keep it there, then having an opt-in / opt-out is good.

But with regard to classes, it makes more sense to have a modifier class on the breadcrumb-item rather than the breadcrumb. For me, I would already be looking for the last breadcrumb-item anyway when debugging this CSS. So, I would just keep this PR the way it is.